dlt/normalize/items_normalizers.py

```diff
@@ -49,20 +51,24 @@ def __call__(
 
 
 class JsonLItemsNormalizer(ItemsNormalizer):
+    @with_plugins()
```

Comment:
you need to pass plugins to normalizer in the same way we pass `DestinationCapabilities`. I just wonder if they are pickable

Reply:
this may run in a process that will be spawned. You cannot expect that, for example, pipeline instance is still present. list of plugins must be self contained. and best if it is stateless (probably impossible)